bundles/org.openhab.binding.tibber/src/main/resources/OH-INF/update/instructions.xml

```diff
@@ -87,6 +87,11 @@
 				<label>Peak Consumption</label>
 				<description>Peak power consumption since midnight in watts</description>
 			</add-channel>
+			<add-channel id="average-consumption" groupIds="live">
+				<type>tibber:power</type>
+				<label>Average Consumption</label>
+				<description>Average power consumption since midnight in watts</description>
+			</add-channel>
```

Comment:
This is not allowed. Please create a new instruction set for target version 4, and bump the `thingTypeVersion` property to 4 as well. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#updating-thing-types